schemas/json/font-collection.json

```diff
@@ -0,0 +1,45 @@
+{
+	"title": "JSON schema for WordPress Font Collection",
+	"$schema": "http://json-schema.org/draft-04/schema#",
+	"type": "object",
+	"properties": {
+		"fontFamilies": {
```

Comment:
Does it make sense for this to be `font_families` (snake cased) instead, since we're outside of theme.json settings and this will be used in PHP?

Reply:
Yep, it makes sense

Reply:
changed :+1: